packages/framework/store/src/store/doc/doc.ts

```diff
@@ -507,9 +508,9 @@ export class Doc {
   }
 
   deleteBlock(
-    model: BlockModel,
+    model: DraftModel,
     options: {
-      bringChildrenTo?: BlockModel;
+      bringChildrenTo?: DraftModel;
```

Comment:
I think it's reasonable; the function does not use that many fields. It might be more appropriate to change it to Pick<BlockModel, ‘id’>, but it's not necessary. I can change it back

Reply:
No, I mean. We probably shouldn't use delete block on the draft model because it's doing an unnecessary delete operation on the backend. It might trigger other bugs.